### PR TITLE
Docs: link pages with TLS

### PR DESCRIPTION
**Pull request by ennru**
## Purpose

Prefer links with TLS over unencrypted HTTP.

## References

https://github.com/akka/akka.github.com/issues/555

### CODE REVIEW

File: akka-docs/src/main/paradox/persistence-query.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ it may be more efficient or interesting to query it (instead of the source event
 
 ### Materialize view to Reactive Streams compatible datastore
 
-If the read datastore exposes a [Reactive Streams](http://reactive-streams.org) interface then implementing a simple projection
+If the read datastore exposes a [Reactive Streams](https://reactive-streams.org) interface then implementing a simple projection
```
Comment (raboof) [inline akka-docs/src/main/paradox/persistence-query.md]: that one doesn't actually work for me

**Review by raboof — APPROVED** [review]
Weird that we were still missing so many, :+1: update. The reactive-streams.org doesn't work for me though...

File: akka-docs/src/main/paradox/persistence-query.md (hunk 1 above)
Comment (ennru) [reply akka-docs/src/main/paradox/persistence-query.md]: Thank you for catching this.

**Comment by ennru** [comment]
I sneaked in a some other link fixes.

**Comment by raboof** [comment]
failure was #27933

**Comment by raboof** [comment]
PLS BUILD

**Review by johanandren — APPROVED** [review]
LGTM

**Pull request merged**